cmd/meroxa/root/apps/logs.go

```diff
@@ -49,13 +45,7 @@ type Logs struct {
 }
 
 type applicationLogsClient interface {
-	GetApplication(ctx context.Context, nameOrUUID string) (*meroxa.Application, error)
-	GetConnectorByNameOrID(ctx context.Context, nameOrID string) (*meroxa.Connector, error)
-	GetConnectorLogs(ctx context.Context, nameOrID string) (*http.Response, error)
-	GetFunction(ctx context.Context, nameOrUUID string) (*meroxa.Function, error)
-	GetFunctionLogs(ctx context.Context, nameOrUUID string) (*http.Response, error)
-	GetLatestDeployment(ctx context.Context, appName string) (*meroxa.Deployment, error)
-	GetResourceByNameOrID(ctx context.Context, nameOrID string) (*meroxa.Resource, error)
+	GetApplicationLogs(ctx context.Context, nameOrUUID string) (*meroxa.ApplicationLogs, error)
```

Comment:
The signature in [the meroxa-go PR](https://github.com/meroxa/meroxa-go/pull/146/files#diff-b0115f9cb6f4166ffbb2be0db889d1b0e6e44d66b95d8a8f6ac7a5aab2548e66R224) is slightly different: `name` as opposed to `nameOrUUID` - seems like we need to adjust that other PR to support `nameOrUUID`, right?

Reply:
parameters in Interface signatures don't even need names, just types, so this can be simplified.  but yes, it's nameOrUUID

Reply:
Yup, the interface implications are clear to me. I was specifically concerned about passing in a UUID as opposed to a name and having that turn into an error. I see that you updated it on the meroxa-go PR. Thanks!